### PR TITLE
add large Co-Processing Unit

### PR DESCRIPTION
**Pull request by asdflj**
add large Co-Processing Unit

### CODE REVIEW

**Comment by repo-alt** [comment]
Shouldn't you also change TileCraftingTile::getItemFromTile() to get correct visual representation in the network tool? (Actually I'm not sure that the advanced co-processors are shown there, so maybe not)

**Comment by asdflj** [comment]
> Shouldn't you also change TileCraftingTile::getItemFromTile() to get correct visual representation in the network tool? (Actually I'm not sure that the advanced co-processors are shown there, so maybe not)

fixed

File: src/main/java/appeng/tile/crafting/TileCraftingTile.java [hunk 1]
```diff
@@ -99,9 +108,20 @@ public boolean isAccelerator() {
...
+            return ACCELERATOR_SCALE_FACTOR;
+        }
+    }
+
     public int acceleratorValue() {
         int meta = this.worldObj.getBlockMetadata(this.xCoord, this.yCoord, this.zCoord) & 3;
-        return (int) Math.pow(ACCELERATOR_SCALE_FACTOR, meta - 1);
+        return (int) Math.pow(
+                this.getAcceleratorScaleFactor(
+                        this.worldObj.getTileEntity(this.xCoord, this.yCoord, this.zCoord).getBlockType()),
```
Comment (eigenraven) [inline src/main/java/appeng/tile/crafting/TileCraftingTile.java:123]: Do `worldObj.getBlock` instead of `worldObj.getTileEntity.getBlockType`

Comment (asdflj) [reply src/main/java/appeng/tile/crafting/TileCraftingTile.java]: Thank you for your advice

Comment (asdflj) [reply src/main/java/appeng/tile/crafting/TileCraftingTile.java]: fixed

**Review by eigenraven — APPROVED** [review]

**Comment by asdflj** [comment]
i found some bug so i need a litte bit time to fix

**Comment by asdflj** [comment]
fixed recipe error and fixed `TileCraftingTile::acceleratorValue()` caculate error

**Pull request merged**